package/package_pico_index.template.json

```diff
@@ -317,6 +317,9 @@
                   {
                      "name": "SparkFun XRP Controller (Beta)"
                   },
+                  {
```

Comment:
Should we remove the other variant?  `SparkFun XRP Controller (Beta)`?

Reply:
No, please keep both. This is for a new version of the same product, but we're planning to maintain support for the old board for a while.